.github/workflows/frontend-remove-unused-components.yml

```diff
@@ -0,0 +1,44 @@
+name: Remove Unused Svelte Files
+
+on:
+  schedule:
+    - cron: '30 3 * * MON'
+  workflow_dispatch:
+
+jobs:
+  check-unused-files:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Prepare
+        uses: ./.github/actions/prepare
+
+      - name: Remove unused files
+        run: ./scripts/check.unused.sh --remove-files
+        continue-on-error: true
```

Comment:
Is that, `continue-on-error: true` correct?

Reply:
This is due to the fact that I return `process.exit(1)` in case of unused files, to continue the workflow. We can remove that return code (or return it only when the files are not removed).

Reply:
AAAAAH gotcha! Shouldn't it return `process.exit(0)` if there is no unused files? If I said the contrary in a previous review, sorry for that.

Reply:
what do you think if I add:

```javascript
} else {
  console.log(...);
  potentialUnusedFiles.forEach((file) => {
    console.log(`${RED}Unused Svelte file: ${file}${NC}`);
    if (REMOVE_FILES) {
      unlinkSync(file);
    }
  });
  
  if (REMOVE_FILES) {
    await main();
  }
  
  process.exit(1);
}
```

it will loop until all unused files are removed (maybe caused by the removal of another one), AND it will exit with 0, so we don't need the `continue-on-error`

Reply:
what about something like following:

```
const print = async (file) => console.log(`${RED}Unused Svelte file: ${file}${NC}`);
const remove = async (file) => {
    await print(file);
    unlinkSync(file);
    console.log(`${GREEN}Removed: ${file}${NC}`
}
		
const fn = REMOVE_FILES ? remove : print;
		
await Promise.allSettled(potentialUnusedFiles.map(fn));
```

Reply:
on it!

But ultimately, I remove the `process.exit(1)` from there, right?

Reply:
I lost track. Let's just make it works ;)